examples/asr/speech_to_text_eval.py

```diff
@@ -122,8 +125,12 @@ def main(cfg: EvaluationConfig):
                 invalid_manifest = True
                 break
 
-            ground_truth_text.append(data['text'])
-            predicted_text.append(data['pred_text'])
+            target_text = separate_punctuation(data['text'])
```

Comment:
Do all this only if user sets flag for it, otherwise do earlier logic.

Reply:
Done